### ISSUE TITLE
Make telescope togglebar and allow switching windows

### ISSUE THREAD

**Issue by gsalvatella** [nvim-telescope/telescope.nvim#2265]
**Is your feature request related to a problem? Please describe.**
When I'm in "search mode", I'd like to have the finder open and switch to the document and back to the finder at will, instead of having to cancel the search and start a new search afresh.

**Describe the solution you'd like**
After opening telescope for a search (non-floating windows only), I can go back and forth to the other buffer windows with the standard bindings (.e.g `<c-w>i/j/h/l`). The telescope search keeps the search string, active element and preview where I left it. When I switch back to it I can continue the search.

**Describe alternatives you've considered**
The alternative is simply cancelling the search with `<Esc>` to get back to your active buffer, do some work and then open a search again.

**Additional context**
This could be implemented as two different features:

1. Add a `remember` option. This would make telescope keep the previous search state where you left it. When telescope is opened again, the search string, active element and preview are still present.
1. Add a `sticky` option. This would treat the telescope window like any other window and the standard window bindings would apply. Pressing `<Esc>` inside the search window would close it as usual.

**Comment by zdleaf** [comment]
The `remember` option can already be done with `:Telescope resume`

`sticky` option would be good though - open results in horizontal/vertical split and keep it there while working in the other split

**Comment by gsalvatella** [comment]
I was not aware yet of the `resume` command. Thank you for sharing it!

**Comment by zdleaf** [comment]
No problem. I'd still like to see the second part. The way I see it is opening or moving Telescope into a normal horizontal/vertical split window instead of a floating window. That way it's always visible and is easily switched back and forth to amend the search term etc.

**Comment by gsalvatella** [comment]
Yes, that was the idea. The window should not disappear after the search-and-jump action.

**Comment by jayywolff** [comment]
This feature request basically sounds like exactly what [ack.vim](https://github.com/mileszs/ack.vim) provides.

Im late to the party switching from vim -> neovim now and i love telescope but the lack of this feature as you described perfectly is the one big gripe i have with find_files. Would be great to get the awesome UI of telescope to work like this without needing to manually push entries into a quickfix window, opening a entry, and then manually opening the quickfix window afterwards, (so many button clicks 🫠 ) so for now, I still have to use ack.vim. Check it out, it does exactly what you need
